cse/pipeline/Pipeline.py

```diff
@@ -1,6 +1,6 @@
 import abc
 
-class Pipeline:
+class Pipeline(object):
```

Comment:
Why inherit the object class?

Reply:
> In Python 3, classes extend object implicitly, whether you say so yourself or not.
>
> In Python 2, there's old-style and new-style classes. To signal a class is new-style, you have to inherit explicitly from object. If not, the old-style implementation is used.
see https://docs.python.org/release/2.5.2/ref/node33.html

Added for backwards compatibility to python2. This is needed for this PR, because getting the right class name for the handlers requires new-style classes

Reply:
I wanted to have consistent class declarations, so I changed all classes to inherit from object (be new-style classes in python2 as well)